omas/omas_sample.py

```diff
@@ -729,6 +733,7 @@ def nbi(ods, nunits=2, ntimes=6):
     times = numpy.linspace(0, 1, ntimes)
     ones = numpy.ones(ntimes)
 
+    ods['nbi.time'] = times
```

Comment:
@jmcclena `ods.plot_nbi_summary()` failed without this. This line made it work on my machine, but did I do it right?